packages/date-picker/src/vaadin-date-picker-overlay-content.js

```diff
@@ -765,6 +768,38 @@ class DatePickerOverlayContent extends ControllerMixin(ThemableMixin(DirMixin(Po
     }
   }
 
+  _onTabKeyDown(event, section) {
+    // Stop propagation to avoid focus-trap
+    // listener when used in a modal dialog.
+    event.stopPropagation();
+
+    switch (section) {
+      case 'calendar':
+        if (event.shiftKey) {
```

Comment:
This branch doesn't seem to be covered by the date-picker tests, but it is covered by the integration tests. What is the general rule for such cases? My expectation is that it should be also covered by the package's tests since this is quite a basic functionality. What do you think?

Reply:
Yes, I agree it should be covered by the component's tests as well. I will add a corresponding test there.

Reply:
Added the missing test to the component.